test/test_binary_ufuncs.py

```diff
@@ -163,6 +163,30 @@ def test_not_broadcastable(self, device, dtype, op):
             )
             raise AssertionError(msg)
 
+    @ops([op for op in binary_ufuncs if op.ref])
+    def test_reference_numerics_normal(self, device, dtype, op):
+        def to_numpy(t):
+            dtype = t.dtype if t.dtype in torch_to_numpy_dtype_dict else torch.get_default_dtype()
+            return t.cpu().to(dtype).numpy()
+
+        for shape in [
+            (812,),
+            (1029, 917),
+            *((),) * 812,
+            (0,),
+            (0, 3, 3),
+            (1, 0, 5),
+            (6, 0, 0, 0),
+            (3, 0, 1, 0),
+        ]:
```

Comment:
Except for the values, this is exactly what is tested in the reference tests for unary ufunc ops. Due to the sheer number of test cases, this introduces a significant slow down if we port more `OpInfo`'s to `BinaryUfuncOpInfo`'s (currently there are only 6). We should discuss this.